homeassistant/components/hue/const.py

```diff
@@ -12,6 +12,6 @@
 DEFAULT_ALLOW_UNREACHABLE = False
 
 CONF_ALLOW_HUE_GROUPS = "allow_hue_groups"
-DEFAULT_ALLOW_HUE_GROUPS = True
+DEFAULT_ALLOW_HUE_GROUPS = False
```

Comment:
How is this going to impact existing users? I think we don't store it in options right?

Reply:
If they have never saved options I think it will use the new default

Reply:
It looks like we migrate it to options if it was in data but the config flow doesn't actually put it in data.  Maybe if it was imported from yaml?

Reply:
Yeah, it was for importing from YAML.